packages/react-magma-dom/src/components/Dropdown/DropdownMenuItem.tsx

```diff
@@ -71,13 +94,17 @@ const StyledItem = styled.div<{
   disabled?: boolean;
   isFixedWidth?: boolean;
   isInactive?: boolean;
+  isInverse?: boolean;
   value?: string | number;
 }>`
   ${MenuItemStyles}
 `;
 
-export const IconWrapper = styled.span`
-  color: ${props => props.theme.colors.neutral03};
+export const IconWrapper = styled.span<{ isInverse?: boolean }>`
+  color: ${props =>
+    props.isInverse
+      ? props.theme.colors.focusInverse
```

Comment:
Can we use `info200` instead of `focusInverse` here, since this is not referring to the focus?

Reply:
Actually, I think this should probably the white?